apis/stardust/api-spec/spec.json

```diff
@@ -781,6 +781,12 @@
                   "defaultValue": true,
                   "type": "boolean"
                 },
+                "toolbar": {
```

Comment:
Related to [PR](https://github.com/qlik-oss/nebula.js/pull/741/files#diff-ed88090db137d093db3797463eeca7b645c1bc1384a08894f3d99dc987c49e7eR344)